include/clap/ext/draft/undo.h

```diff
@@ -76,25 +77,43 @@ typedef struct clap_plugin_undo {
    bool(CLAP_ABI *can_use_delta_format_version)(const clap_plugin_t *plugin,
```

Comment:
should it be allowed to call this if `clap_undo_delta_properties_t::are_deltas_persistent` is false?

Reply:
🤔 maybe not

Reply:
Adding this requirement could be correct, but I don't see it necessary.
I prefer to not add this requirement to the spec, helpers, ...